### PR TITLE
Text changes

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description

supports tickets:
- [https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1897](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1897)
  Rename 'Licensees' field to 'Organizations'.

- [https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1903](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1903)
  only implements the changes highlighted in word document

- and temporary fix to prevent dev crashing from keycloak admin refresh token issue. possibly due to RH sso upgrade.
(see second commit)

## Types of changes

New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

.. with the db migration file, i am adding in three extra fields to the mines_os_submission_attestation table.
it seems with knex that (unless youre using MySQL) you have to add them to the end of the table (after the timestamp fields) which is a bit ugly.

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://comfort-dev.apps.silver.devops.gov.bc.ca/pr-161

File: app/src/forms/attestations/minesoperatorscreening/models/form.js [hunk 1]
```diff
@@ -106,6 +106,23 @@ class Attestation extends AttestationModels.AttestationTransportation {
     return PREFIX;
   }
 
+  static get jsonSchema() {
+    const schemaProperties = {...AttestationModels.AttestationTransportationSchema.properties};
+    // add new properties
+    schemaProperties.transportationFaceCovering = {type: 'boolean'};
+    schemaProperties. distancingStayInAccom = {type: 'boolean'};
+    schemaProperties.trainingDailyChecks = {type: 'boolean'};
```
Comment (loneil) [inline app/src/forms/attestations/minesoperatorscreening/models/form.js:114]: Not really sure what this is for? These are fields in the  submission_attestation table so should be defined in the same place as all the other ones. Any reason the 3 new fields are singled out in this new method?

**Comment by TimCsaky** [comment]
just to summarise:
the changes from Brittany for the mines form only include the 3 extra checkboxes and an additional supporting document on the landing page. I just noticed the supporting documents are listed on the submission confirmation page.. so i'll add the extra one now.

File: app/src/forms/attestations/minesoperatorscreening/models/form.js (hunk 1 above)
Comment (TimCsaky) [reply app/src/forms/attestations/minesoperatorscreening/models/form.js:114]: Yes, it does look a bit strange but it seemed to follow the convention already there. 

Any changes to the models that arent common to all forms are in app/src/forms/<form>/models/form.js
the agriseafood and mines forms have some additional properties by extending the AttestationTransportation model as well as adding the sleeping related ones.

If i add these three new properties to the 'base' attestation model, then they are required for all 3 forms.

i think youre right, i might be able to do this a cleaner way

Comment (loneil) [reply app/src/forms/attestations/minesoperatorscreening/models/form.js:114]: 👍

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://comfort-dev.apps.silver.devops.gov.bc.ca/pr-161

**Review by loneil — APPROVED** [review]
Looking good

**Review by jujaga — APPROVED** [review]

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://comfort-dev.apps.silver.devops.gov.bc.ca/pr-161

**Pull request merged**